scripts/install_docker.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+set -ex
+
+curl -sSL https://get.docker.com/ubuntu/ | sh
```

Comment:
ok. When I run your command on a VBox, I get this error:

E: Unable to locate package lxc-docker

Looks like lxc-docker us a prereq.